hcm:

  start_time: 0 # time we wait for getting motor updates before leaving startup state
  motor_off_time: 30000000 # time of no use or updates when the hcm goes to soft off
  imu_timeout_duration: 0.1 # time without messages from the IMU when error is produced [s]
  motor_timeout_duration: 0.1 # time without messages from the IMU when error is produced [s]
  anim_server_wait_time: 10 # time the hcm waits for animation server

  stand_up_active: false # enables the robot to stand up automatically
  falling_active: false # enables the check for falling and coresponding counter meassurements
  #Grenzwerte sch채rfer --> fruehere Reaktion aber haeufigere Fehlinterpretation, zb. beim Laufen
  #Grenzwerte sanfter --> zu spaete Raktion
  threshold_gyro_y_front: 2.5 # > gyroY
  threshold_gyro_x_side: 2.5 # > gyroX
  falling_threshold_orientation_left_right: 30 # > Point of no return in degrees
  falling_threshold_orientation_front_back: 30 # < Point of no return in degrees
  falling_new_front: 32
  falling_new_back: -6
  falling_new_left: -18
  falling_new_right: 18

  animations:
    walkready: "walkready" 
    falling: "falling"
    stand_up_front: "stand_up_front"
    stand_up_back: "stand_up_back"
    sit_down: "walkready"
    penalty: "walkready"
    motor_off: "walkready"
    init: "init"
    walkready_pose: {"HeadPan": 0, 
    "HeadTilt": 0, 
    "LElbow": 45, 
    "LShoulderPitch": 0, 
    "LShoulderRoll": 0, 
    "RElbow": -45, 
    "RShoulderPitch": 0, 
    "RShoulderRoll": 0, "LHipYaw" : -0.9201349940933852,
    "LHipRoll" : 2.442961025290042,
    "LHipPitch" : 50.31854042205566,
    "LKnee" : -58.438368356309766,
    "LAnklePitch" : -28.765928610673875,
    "LAnkleRoll" : 2.6104015763666046,
    "RHipYaw" : 0.9201343746780442,
    "RHipRoll" : -2.4429603552214245,
    "RHipPitch" : -50.31855053488634,
    "RKnee" : 58.43838767715415,
    "RAnklePitch" : 28.76593741901185,
    "RAnkleRoll" : -2.610399717389955}
    walkready_pose_threshold: 5 # [deg]

  falling:

    #falling animations
    "falling_front": "falling_front"
    "falling_back": "falling_back"
    "falling_left": "falling_left"
    "falling_right": "falling_right"

    "Wolfgang":

      #Grenzwerte m체ssen auf h채rterem boden auch h채rter eingestellt werden,
      #dies erfolgt mit dem boden_koeffizient
      "ground_coefficient": 1.0  #WEICH(kunstrasen Spielfeld)= 1.0 , HART(teppich Spielfeld)= 1.25

      #false = motoren werden bei fallerkennung abgeschaltet
      "dyn_falling_active": true
